src/main/java/org/folio/dcb/service/impl/CustomTenantService.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.dcb.service.impl;
+
+import lombok.extern.log4j.Log4j2;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.liquibase.FolioSpringLiquibase;
+import org.folio.spring.service.PrepareSystemUserService;
+import org.folio.spring.service.TenantService;
+import org.folio.tenant.domain.dto.TenantAttributes;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+@Log4j2
+@Service
+@Primary
+@Lazy
```

Comment:
is there any way to get rid of this @Lazy annotation. This may lead to unexpected behaviour sometime and impact on performence , proxy , transaction management etc.

Reply:
@SinghAdes take a look a bit deeper into the meaning of this anno via JavaDoc at least. Its value is pretty straightforward and doesn't lead to unexpected behavior.
Feel free to reach me directly for a detailed discussion.
Thx